Sources/PT.PM.Common/Nodes/Tokens/Literals/Literal.cs

```diff
@@ -1,5 +1,17 @@
-﻿namespace PT.PM.Common.Nodes.Tokens.Literals
+﻿using MessagePack;
+
+namespace PT.PM.Common.Nodes.Tokens.Literals
 {
+    [MessagePackObject]
+    [Union((int)NodeType.BinaryOperatorLiteral, typeof(BinaryOperatorLiteral))]
```

Comment:
Is it necessary to duplicate the `Union` attribute at all stages of class inheritance?

Reply:
Unfortunately yes. Also, there is a special test `Check_MessagePackTypes` for checking if all abstract classes mapped to concrete ones.